.github/workflows/publish.yaml

```diff
@@ -0,0 +1,62 @@
+name: 🚀 Publish assets to NPM
+
+on:
+    release:
+        types: [ published ]
+
+jobs:
+    publish-prime:
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: ./src/Prime/Resources
+        permissions:
+            contents: read
+            id-token: write
+        steps:
+            - uses: actions/checkout@v4
+
+            # Setup .npmrc file to publish to npm
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: '20.x'
+                  registry-url: 'https://registry.npmjs.org'
+
+            - run: npm ci
+            - run: npm publish --provenance --access=restricted
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
note to self: 

Created a NPM account and added `NPM_TOKEN` to action secrets in repo already.